src/client/SConscript

```diff
@@ -7,5 +7,9 @@ def scons():
     SConscript('api/SConscript')
```

Comment:
The commit message should be fixed as per our standards (i.e. DAOS ticket, component name, ...).